packages/fxa-auth-server/config/index.ts

```diff
@@ -2241,42 +2241,42 @@ const convictConf = convict({
   },
   twilio: {
     credentialMode: {
-      default: 'test',
-      doc: 'Which credential set to use. Options are test, default, or apiKeys.',
+      default: '',
+      doc: 'Which credential set to use. To enable twilio, options are "", "test", "default", or "apiKeys".',
```

Comment:
"" option wouldn't enable twilio.

```suggestion
      doc: 'Which credential set to use. Options are "", "test", "default", or "apiKeys".',
```